examples/node-esm/__integration-tests__/esm.spec.mjs

```diff
@@ -0,0 +1,18 @@
+import { yoga } from '../yoga.mjs'
+
+describe('Node ESM', () => {
+  it('should work', async () => {
+    const response = await yoga.fetch('https://yoga/graphql', {
+      method: 'POST',
```

Comment:
Why Yoga fetch? Aren't we testing Node here?

Reply:
it is more about running the tests in esm i think

Reply:
I think @saihaj created this test?

Reply:
I don't recall this 😂